pom.xml

```diff
@@ -558,6 +558,15 @@
             <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Can you add a comment before the plugin config to explain: 
- Needed to bypass the encoding of the `passphrase` when task is executed by a job and not a human
- Syntax could change with future releases of the plugin (>= 3.0.0)

Reply:
Done.